### DISCUSSION TITLE
Thanks for setting this up!

### DISCUSSION THREAD

**Discussion by jakemelt** [CommandPost/FCPCafe#105] [General Discussions]
Thanks @latenitefilms, for setting this up. It's a cool approach to collaborating on FCP info, a refreshing break from social media stuff.

- jake

**Comment by latenitefilms** [comment]
Thanks Jake! Hopefully it will grow and expand over the coming weeks, months and years! I'm certainly excited about it - as it gives me a convenient place to share my knowledge with the world!

**Comment by hexeric** [comment]
yes, love it too! already found some news/infos on here that i did not anywhere else (e.g. the free BCC plugin) – and concerning the latest news...i am so done with communities of other NLEs – the FCP bunch is one of a kind!

**Comment by ennoedits** [comment]
I rarely ever do social media, so this will be a new go-to destination for me. Thank you for making this happen! Love the format.
